modules/monitoring-accessing-the-targeting-ui.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="monitoring-accessing-the-targeting-ui_{context}"]
+= Accessing the Targeting UI in the Administrator perspective
+
+You can view the Targeting UI in the Administrator perspective in the {product-title} web console.
+
+* In the *Administrator* perspective, select *Observe* -> *Targets*. 
+The *Metrics Targets* page opens with a list of all HTTP endpoint targets that are being scraped for metrics.
```

Comment:
we also have HTTPS endpoint
change to HTTP/HTTPS is better

Reply:
I removed "HTTP endpoint" and replaced it with "service endpoint". Please lmk if this is OK.